src/storekit.cs

```diff
@@ -239,10 +259,12 @@ interface SKProduct {
 		string ProductIdentifier { get; }
 
 #if MONOMAC
+		[NoWatch] // not necessary b/c #if MONOMAC, so this will not even compile for ios/watch/tvos?
 		[Deprecated (PlatformName.MacOSX, 10,15, message: "Use 'IsDownloadable' instead.")]
 		[Export ("downloadable")]
 		bool Downloadable { get; }
 #elif !XAMCORE_4_0
+		[NoWatch] // IS THIS RIGHT?
```

Comment:
Not sure about this one, but since it is `Obsolete` it seems like we would not want users to have access to it starting in `[Watch (6, 2)]`?

Reply:
no, it's not needed

some times when Apple enable an existing framework in a new platform then bring everything, including deprecated API :(
Since this is replaced by something newer/better, that is always available on the new platform, then we can reduce warnings and let intellisense provide better choice to the developers

if xtro complains (because it's in the header) then add a line in watchOS-StoreKit.ignore